docs/csharp/programming-guide/classes-and-structs/codesnippet/CSharp/interfaces_1.cs

```diff
@@ -1,4 +1,4 @@
-interface IEquatable<T>
+    interface IEquatable<T>
     {
         bool Equals(T obj);
     }
```

Comment:
I think the interface should not be indented at all. In other words, I think 4 spaces of indentation should be removed from lines 2-4, instead1.